REQUIRE

```diff
@@ -1,7 +1,8 @@
 julia 0.5
-DataStructures 0.4.3
+DataStructures 0.5.0
 DataFrames 0.9
-NullableArrays 0.0.10
-CategoricalArrays 0.0.6
+NullableArrays 0.1.0
+CategoricalArrays 0.1.0
+AxisArrays 0.0.6
```

Comment:
R named arrays don't have anything really specific. Names are always strings, indexing by name is supported (but only by putting the index in the position corresponding to the dimension). Names are propagated by some operations like element-wise ops and reductions (which isn't supported by AxisArrays yet IIRC). The way they are printed is similar to the `NamedArray` example above. I was afraid the way `AxisArrays` are printed was a design decision, but if you would be OK with something more like `NamedArray` it would be great.

(Then I'm not a fan of a name, I wouldn't have considered dimension names as axes, but I guess that's a minor point. :-)